hotel/templates/registration/review.html

```diff
@@ -0,0 +1,9 @@
+{% extends "registration/review-base.html" %}
+
+{# This is kind of a hack, it defines a way that admins can force marking someone as hotel eligible. #}
+{# When we switch from Django to Jinja2, this can be injected into the main registration form #}
```

Comment:
I'm not familiar with this comment syntax for Django.  I just googled for it but it looks like
https://docs.djangoproject.com/en/1.10/ref/templates/builtins/#comment
is still the way you're supposed to leave comments?

Still, what you have here presumably works since you've tested it, so assuming that this actually does what we want, then I'm cool with it.